File: igor/Problema0239/problema0239.java
import java.util.Scanner;
public class problema0239 {

    static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {

        System.out.println("Escreve numero base 8: ");
        String base8= input.nextLine();
        while(!(base8.equals("*"))) {
            System.out.println("Base 8: "+base8);
            String base10 = Integer.toString(Integer.parseInt(base8, 8), 10);
            System.out.println("Base 10: "+base10);

            System.out.println("Escreve numero base 8: ");
            base8= input.nextLine();
        }
    }
}

File: igor/Problema0239/problema0239.py
base8 = input("Escreve numero base 8: ")

while base8 != "*":
    print("Base 8: ",base8)
    base8 = str(int(base8,8))

    base10 = int(base8, 10)
    print("Base 10: ",base10)

    base8 = input("Escreve numero base 8: ")